Dockerfile-etl

```diff
@@ -6,13 +6,15 @@ COPY . /isaac-api
 RUN mvn package -Dmaven.test.skip=true -P etl
 
 # create clean jetty docker container
-FROM jetty:9.4.48-jre8-openjdk as server
+FROM jetty:9.4.48-jdk11-eclipse-temurin as server
 USER root
 COPY --from=target /isaac-api/target/isaac-api.war /var/lib/jetty/webapps/isaac-api.war
 RUN chmod 755 /var/lib/jetty/webapps/*
 RUN chown jetty /var/lib/jetty/webapps/*
 
-#RUN sed -i -e 's#dev/random#dev/./urandom#g' $JAVA_HOME/jre/lib/security/java.security
+# The Jetty Docker image provides a version of JavaMail incompatible with JDK11,
+# remove the bundled version:
+RUN rm -r /usr/local/jetty/lib/mail
```

Comment:
Just making a note that we will probably have to remove this when we update Jetty (which will likely include all sorts of other issues moving to the `jakarta` namespace).